geode-junit/src/main/java/org/apache/geode/codeAnalysis/AnalyzeSerializablesJUnitTestBase.java

```diff
@@ -305,6 +297,10 @@ private void serializeAndDeserializeSanctionedObject(Object object) throws Excep
     BufferDataOutputStream outputStream = new BufferDataOutputStream(KnownVersion.CURRENT);
     try {
       serializeObject(object, outputStream);
+    } catch (RemoteException e) {
+      System.out.println(object.getClass().getName() +
+          " is a java.rmi.server.RemoteObject which is not supported by AnalyzeSerializables");
+      return;
```

Comment:
Note: geode-dunit uses RMI. Now that geode-dunit uses sanctioned serializables, a couple RMI classes are fed into this test. Supporting java.rmi.server.RemoteObject would require extra finesse that this test doesn't do and it's not worth implementing it for a test framework.